distribution/docs/src/main/resources/content/_integrating/_endpoints/catalog-rest-endpoint.adoc

```diff
@@ -3,291 +3,320 @@
+        "id": "3a59483ba44e403a9f0044580343007e",
+        "metacard-type": "${branding-lowercase}.metacard",
+        "title": "Test REST Metacard",
+        "source-id": "${branding-lowercase}.distribution",
+        "metacard.created": "${timestamp}"
+    }
+}
 ----
-// https://github.com/codice/ddf/issues/4374
-//.Create Success Response Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
-
-.Create Error Response Examples
-If content or metadata is not ingested successfully, check for these error messages.
 
-.Malformed XML Response
-If the XML being ingested is not well-formed, an HTTP 400 is returned and the following response body is returned and
-the specific error is logged in the error log.
+To retrieve a metacard from a specific federated source, add `sources/<SOURCE_ID>` to the URL.
 
-[source,html]
+.Federated Read Request URL
+[source,http]
 ----
-<pre>Error while storing entry in catalog: </pre>
+https://<FQDN>:<PORT>/services/catalog/sources/<sourceId>/<metacardId>?transform=<TRANSFORMER_ID>
 ----
 
-.Request with Unknown Schema
-If ingest is attempted with a schema that is unknown, unsupported, or not configured by the endpoint,
-${branding} creates a generic resource metacard with the provided XML as content for the `metadata` XML field in the metacard.
+.[[_catalog_rest_retrieve_example]]Resource Retrieval Operation Examples
+To retrieve the resource associated with a metacard, use the `resource` transformer with the `GET` request.
 
-.Request with Missing XML Prologue
-If a request with a missing XML prologue is sent to the Catalog ReST endpoint,
-the metacard is created successfully.
+.Retrieve Resource Request URL
+[source,http]
+----
+https://<FQDN>:<PORT>/services/catalog/<metacardId>?transform=resource
+----
 
-.Request with Non-XML Data
-If a request with non-XML data sent to the Catalog ReST endpoint, the metacard will be created and the content will stored in the `metadata` field.
-
-'''
-
-//https://github.com/codice/ddf/issues/4374
-//.[[_catalog_rest_update_example]]Update Operation Examples
-//notes
-//
-//.Update Request Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
-//
-//.Update Response Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
-//'''
-//.[[_catalog_rest_delete_example]]Delete Operation Examples
-//notes
-//
-//.Delete Request Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
-//
-//.Delete Response Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
+See <<{architecture-prefix}available_metacard_transformers,Metacard Transformers>> for details on metacard transformers.
 
-.[[_catalog_rest_read_example]]Read Operation Examples
+.Read Error Response Examples
+If the metacard is not returned successfully, check for these errors.
 
-The `read` operation can be used to retrieve metadata in different formats.
+.Create Error Response Examples
```

Comment:
✏️ How is this different than the other `.Create Error Response Examples` table? Shouldn't they be merged together?

Reply:
typo. should be `read error response examples`